charts/opentelemetry-operator/kubeconfig

```diff
@@ -0,0 +1,17 @@
+    client-key-data: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcFFJQkFBS0NBUUVBMSt3VTV2MDZuczFVYmNURXJUNndSZHFsVkxERnJLWUlNVjRKZVBjYm1sOG9ZR09DCnpBWUtqNFRxb2hhT0t2TXNRQnlWc0R2ZTZ0c25kcEQ2WW9tQTVoc2w5ZkZFS05GOTVpYzZaVGhJMy9Ld3pHTWUKOEcwdG51UU00RnpJcWxKSDhmbzZYZXhlQmdPRHQ2SW1MNFdUN1JGVVFqblhaM3VLNkg3UU80RXFkRXhjL2hITApRWmRCT3FwbEhUS3p4Z2RkT0MrdnVrUHRqQzRPNzZ4a3V5dm9tV0hlTUhQVUpvSW1LV1JLZFl6RHpTK2tRNHNOCnhyNkQwbkYrMGZLRDBubFdIVkNjRmU3NWlwK1p5K3l6K1h4Qk1kS2VrT1luQXhBdm5DTlM2WjZyYWttTThBRFYKeDNGeEpVMFc2L0VwdnZaMmlWby9hMGUxOHJYSzZPUkJGZ1B0bVFJREFRQUJBb0lCQUJwdTNWUTExZkpRNHJBWgpjWlpmNFVaNzhOZUFXSUs5UksxK2ZKeGhhL0NuQzVOWk8wN3N3bWcxZGwvbmNQb1RFT3k4VXhyakNYUW81aGplCndMaEZheEZBYTVKRE9mdmhRWTltYUFDeU5rR3NQVGFZUDlaaEZ6K25YV2d1RTZhK01lNk9tb3lhL2h1d1hIU2oKbGhjbFBDbStnRmtuS1phU2lqdlNJbjNXWHpNaldGS3c1WmJqamRsVy9TS3Z3L2FwMlNOWWZnMXlOcGdhMWlEVwpiTkdIcWowcEZQMlUwamJ1bnpqWHd3bi8rNHpzTk9EcVRyMGt5U3ZrcXlnWVV4REppaXRJVXhZZk9CbThDcnRhCm5mRUEvY0ZIWDc0a3k0NVFiVzhqYVhKbTRMd003Z3hhUjhsSGY1ekhrNDFYd29jbjVRYTI1M1MwQ2N4WUxuSnAKVDUvMkZqMENnWUVBL3RKOWhZK21yRjZHY3J4QWhTbWlHUmMwUVZtVHErdTh4c2hwVU45Yks1dVM0ZGZnV1F6LwptNzZzdjlDdHd6K1UxWWk3a3VpaXB4R2puUFN4Vm5zMERnYnJsOXBCVmNxd2dJTUViL0VxRjZqWDRiRE9VZ29ZCktoQVdoWUdSSU9kaGQzNGJ2aWNEbjgxbjhrRWRIS1g2MkxOTXF0ZXo3aDBrRjNqV291RnNjWXNDZ1lFQTJPdVEKYnk3Q3BGYTh0Vms2TmM3NTVySDhMeXRjZnZHN0pGelpxZVNLOWxZT2VuVk1ydFN3WTFCOUp6bTQ4eS9GTUNSTApFbHFJeVgya2RybFBmN0ZMNDFPc3M5KzFlMk1qNUY1R1VtZEFDOG0zbWlTRTUybDZWOUlIblZPZkNkUGVXVDJ1CnQxZHlHTThaTE01MXZBVjVNNkJDQ0RsOEJseTdzdWlEdmNFSGVlc0NnWUVBbG5KbUE0WUZqTDBlT2YxbFJubUgKSjdNb2VmRE9HL0hKZWI5M1ZuVVltYzEzVWZtbmprdjdXZE9nb0hzdkxCaU00ZW1oZlNIMmJoY2U5eHNNM0pkTwo2MzB3ZVc0MXB0Wk5rUUpKaTJIaVRTZTFsWjFTb3VNczB5QytlMEk5RWlVcHlXZmVMMDFQZVhqSGh5TGtjamFZCnZIcFhxSUpEekRET1QwRmJ6N3l4enZNQ2dZRUFoNnRGU1pTSVhqcDFpa0xGT25RNkxrUXhndHJ1M2VLWWRsU3cKT2liYmx0Z1o5Qk03STdtWW5IVHg0YitqWWs0RldXZkpvREFKajJQS29ncWlveTJqVmdtMllVWFFmdHh4elZuOQpSM3FXUHdrcGtQbTRPZmZwY2RYRnQydGVjOG9VTGVKN1RPWWZVVTd0NHUwNkM1K3N2MUpER2x2QnEwVUNqQklLCm0zcUlENE1DZ1lFQTI0RTdLRDcwVHJsSWh1cGpzb1h2Z0h4eThXdjVHYStRSVMzOWRkOHVKTWxwREJhRzg3RkQKWFFSTzkxajlhWjFrMENOcnAzTk5XektFWlpQNnBTRW5zVnVCd0xNZ1FOclpualh6WEcxVVNYWmxQeTJyejVXbwpCVExMeFJhcVBqdlEzZkE3L2svdkdFTFFFTTNWNGRlbWVxekppUEkwdHlvR1BJMUZia2hLaGJrPQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
Why is this file needed?

Reply:
This file is self generated when I run KUTTL tests locally. I removed it. Sorry for the confusion!